tests/Bridge/Elasticsearch/DataProvider/PaginatorTest.php

```diff
@@ -76,6 +76,9 @@ class PaginatorTest extends TestCase
         ],
     ];
 
+    protected $offset = 4;
```

Comment:
Both parameters should be class constants.

Reply:
Ok. I put this params as class constants.